build-push-release-image/action.yml

```
@@ -21,7 +21,7 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: toptal/davinci-github-actions/yarn-install@v4.8.2
+    - uses: toptal/davinci-github-actions/yarn-install@v4.8.3
```

you might use the latest version 🙇 
```suggestion
    - uses: toptal/davinci-github-actions/yarn-install@v4.8.5
```

---

To close this ticket, I'll move forward as is for now if it's ok :slightly_smiling_face: